host_core/lib/host_core/application.ex

```diff
@@ -62,72 +70,41 @@ defmodule HostCore.Application do
       {HostCore.HeartbeatEmitter, config},
       {HostCore.Providers.ProviderSupervisor, strategy: :one_for_one, name: ProviderRoot},
       {HostCore.Actors.ActorSupervisor, strategy: :one_for_one, name: ActorRoot},
-      {HostCore.Claims.Manager, strategy: :one_for_one, name: ClaimsManager},
-      {HostCore.Linkdefs.Manager, strategy: :one_for_one, name: LinkdefsManager},
-      # Handle advertised link definitions and corresponding queries
-      Supervisor.child_spec(
-        {Gnat.ConsumerSupervisor,
-         %{
-           connection_name: :lattice_nats,
-           module: HostCore.Linkdefs.Server,
-           subscription_topics: [
-             %{topic: "wasmbus.rpc.#{config.lattice_prefix}.linkdefs.put"},
-             %{topic: "wasmbus.rpc.#{config.lattice_prefix}.linkdefs.del"},
-             %{
-               topic: "wasmbus.rpc.#{config.lattice_prefix}.linkdefs.get",
-               queue_group: "wasmbus.rpc.#{config.lattice_prefix}.linkdefs.get"
-             }
-           ]
-         }},
-        id: :linkdefs_consumer_supervisor
-      ),
-      # Handle advertised PK->Claims maps and corresponding queries
+      # Handle lattice control interface requests
       Supervisor.child_spec(
         {Gnat.ConsumerSupervisor,
          %{
-           connection_name: :lattice_nats,
-           module: HostCore.Claims.Server,
+           connection_name: :control_nats,
+           module: HostCore.ControlInterface.Server,
            subscription_topics: [
-             %{topic: "wasmbus.rpc.#{config.lattice_prefix}.claims.put"},
+             %{topic: "wasmbus.ctl.#{config.lattice_prefix}.cmd.#{config.host_key}.*"},
+             %{topic: "wasmbus.ctl.#{config.lattice_prefix}.ping.hosts"},
              %{
-               topic: "wasmbus.rpc.#{config.lattice_prefix}.claims.get",
-               queue_group: "wasmbus.rpc.#{config.lattice_prefix}.claims.get"
-             }
-           ]
-         }},
-        id: :claims_consumer_supervisor
-      ),
-      # Handle advertised OCI->public key reference maps and corresponding queries
-      Supervisor.child_spec(
-        {Gnat.ConsumerSupervisor,
-         %{
-           connection_name: :lattice_nats,
-           module: HostCore.Refmaps.Server,
-           subscription_topics: [
-             %{topic: "wasmbus.rpc.#{config.lattice_prefix}.refmaps.put"},
+               topic: "wasmbus.ctl.#{config.lattice_prefix}.linkdefs.*",
+               queue_group: "wasmbus.ctl.#{config.lattice_prefix}"
+             },
              %{
-               topic: "wasmbus.rpc.#{config.lattice_prefix}.refmaps.get",
-               queue_group: "wasmbus.rpc.#{config.lattice_prefix}.refmaps.get"
-             }
+               topic: "wasmbus.ctl.#{config.lattice_prefix}.get.>",
+               queue_group: "wasmbus.ctl.#{config.lattice_prefix}"
```

Comment:
Are there any implications here for `linkdef` operations and `get` operations to operate on the same queue group?

Reply:
the queue group defines the "psuedo random selection" list from among potential subscribers to a given topic name. By setting the queue group name to what it is, we're basically just guaranteeing that for each topic where appropriate, the subscribers are chosen randomly from the list of all subscribers in the lattice. 

tl;dr this should be fine.

Reply:
Cool that's what I figured 👍🏻